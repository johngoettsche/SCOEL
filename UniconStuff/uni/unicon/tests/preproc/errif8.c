/* test error message: missing #end after #elif */

#if 0
one
#elif 0
two
