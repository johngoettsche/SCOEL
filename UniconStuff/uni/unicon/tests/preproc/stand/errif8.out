pp: File errif8.c; Line 5: no matching #endif for #elif
